lms/templates/verify_student/make_payment_step_ab_testing.underscore

```diff
@@ -0,0 +1,108 @@
+            </div>
+            <p>
+                <% if ( courseModeSlug === 'no-id-professional' || courseModeSlug === 'professional') { %>
+                    <%= _.sprintf(
+                        gettext( "Professional Certificate for %(courseName)s"),{  courseName: courseName }
+                     )%>
+                <% } else { %>
+                    <%= _.sprintf(
+                        gettext( "Verified Certificate for %(courseName)s"),{  courseName: courseName }
+                     )%>
+                <% } %>
+            </p>
+        </div>
+    <% } %>
+
+
+    <% if ( isActive ) { %>
+      <div class="payment-buttons is-ready center">
+        <input type="hidden" name="contribution" value="<%- minPrice %>" />
+        <input type="hidden" name="sku" value="<%- sku %>" />
+        <div class="pay-options">
+          <%
+          // payment buttons will go here
+          %>
+        </div>
+      </div>
+      <div class="border-gray"></div>
+    <% } %>
+
+   </div>
+
+    <% if ( isActive ) { %>
+        <div class="container">
+            <% if ( _.some( requirements, function( isVisible ) { return isVisible; } ) ) { %>
+                <p>
+                    <% if ( verificationDeadline ) { %>
+                      <%- _.sprintf(
+                        gettext( "In addition to payment, you will need to verify your identity (before %(date)s)." ),
+                        { date: verificationDeadline }
+                      ) %>
+                    <% } else  { %>
+                        <%- gettext( "In addition to payment, you will need to verify your identity." ) %>
+                    <% } %>
+                     <%- gettext("For verification, you will need a webcam and a photo ID.") %>
+                </p>
+                <% if ( requirements['photo-id-required'] ) { %>
+                    <div class="photo-requirement user_icon">
+                        <i class="fa fa-user fa-2x" aria-hidden="true"></i>
+                        <h6>
+                            <%- gettext("Your Photo ID") %>
+                        </h6>
+                        <p>
+                            <%- gettext("Cheating rarely happens, but we need make sure you are who you say you are.") %>
+                        </p>
+                    </div>
+                 <% } %>
+
+                <% if ( requirements['webcam-required'] ) { %>
+                    <div class="photo-requirement cam_icon">
+                        <i class="fa fa-video-camera fa-2x" aria-hidden="true"></i>
+                        <h6>
+                            <%- gettext("Enable Your WebCam") %>
+                        </h6>
+                        <p>
+                            <%- gettext("Similar to your ID, we verify that when you take exams, you are who you say you are.") %>
```

Comment:
It might be helpful to catch up offline about this specific line. I don't know if the suggestion of using a smartphone or library computer are necessarily possible in enough cases that it is worth showing to everyone. (nit)